docs/openstack-cloud-controller-manager/using-openstack-cloud-controller-manager.md

```diff
@@ -31,6 +31,8 @@ For more information about cloud-controller-manager, please see:
 
 **NOTE: Now, the openstack-cloud-controller-manager implementation is based on OpenStack Octavia, Neutron-LBaaS has been deprecated in OpenStack since Queens release and no longer maintained in openstack-cloud-controller-manager. So make sure to use Octavia if upgrade to the latest openstack-cloud-controller-manager docker image.**
 
+**NOTE: `topology.kubernetes.io/zone` is used to label node and its value comes from availability zone of the node, according to [label spec](https://kubernetes.io/docs/concepts/overview/working-with-objects/labels/#syntax-and-character-set) it does not support blank (' ') but OpenStack availability zone supports blank. So your OpenStack availability zone must not contain blank otherwise it will lead to node register issue, see [#1379](https://github.com/kubernetes/cloud-provider-openstack/issues/1379) for further information.**
```

Comment:
@jichenjc I would prefer adding in separate Limitations section below. And also , please be explicit , node registration fails if node AZ contains blank in its name.

Reply:
ok, will do